classes/cliTool/CommandLineTool.php

```diff
@@ -120,9 +122,22 @@ private function checkArgsForUsername()
             $adminGroups = Repo::userGroup()->getArrayIdByRoleId(Role::ROLE_ID_SITE_ADMIN);
 
             if (count($adminGroups)) {
-                $groupUsers = Repo::user()->getCollector()
-                    ->filterByUserGroupIds([$adminGroups[0]])
-                    ->getMany();
+                $rows = DB::table('users', 'u')
```

Comment:
I added a new function in the User Repo that ignores the new columns start and end date, and that I then use here, because the new columns start and end date do not exist yet, at the moment when the CLI tool is executed...